src/components/ReportNavBar/styles.scss

```diff
@@ -27,11 +27,17 @@
       margin: 0;
     }
 
-    .view-button {
+    .view-button{
       height: 2em;
       width: 2em;
       cursor: pointer;
       fill: #AAB;
+      &-active{
```

Comment:
This block can be refactored to avoid the repetition, and will as well require some refactoring in the `ReportNavBar\index.js` file